dali/python/nvidia/dali/ops.py

```diff
@@ -1048,27 +1060,30 @@ def _cupy_stream_wrapper(function, *inputs):
 
     @staticmethod
     def _function_wrapper_gpu(batch_processing, function, num_outputs, *dlpack_inputs):
+
         def wrapped_func(*inputs):
             return PythonFunction._cupy_stream_wrapper(function, *inputs)
+
         if batch_processing:
             return PythonFunction.function_wrapper_batch(wrapped_func, num_outputs, cupy.fromDlpack,
                                                          lambda t: t.toDlpack(), *dlpack_inputs)
         else:
-            return PythonFunction.function_wrapper_per_sample(wrapped_func, num_outputs, cupy.fromDlpack,
+            return PythonFunction.function_wrapper_per_sample(wrapped_func, num_outputs,
+                                                              cupy.fromDlpack,
                                                               lambda t: t.toDlpack(),
                                                               *dlpack_inputs)
 
     def __init__(self, function, num_outputs=1, device='cpu', batch_processing=False, **kwargs):
         if device == 'gpu':
             _setup_cupy()
-        func = (lambda *ts: PythonFunction._function_wrapper_cpu(batch_processing, function, num_outputs, *ts))\
-               if device == 'cpu' else \
-               (lambda *ts: PythonFunction._function_wrapper_gpu(batch_processing, function, num_outputs, *ts))
-        super(PythonFunction, self).__init__(impl_name="DLTensorPythonFunctionImpl",
-                                             function=func,
-                                             num_outputs=num_outputs, device=device,
-                                             synchronize_stream=False,
-                                             batch_processing=batch_processing, **kwargs)
+        func = (lambda *ts: PythonFunction._function_wrapper_cpu(
+            batch_processing, function, num_outputs, *ts)) if device == 'cpu' else (
+                lambda *ts: PythonFunction._function_wrapper_gpu(batch_processing, function,
+                                                                 num_outputs, *ts))
```

Comment:
PEP 8 discourages using backslashes to wrap a long line:

> The preferred way of wrapping long lines is by using Python’s implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.

So this is more or less how it should be according to the best practices proposed by PEP 8. I think that for someone accustomed to the PEP 8 standard this new formatting might actually be more readable than the old one.

With that said, I decided to unpack this code a bit and refactored it to be regular `if` `else` statement instead of a short hand notation.